### DISCUSSION TITLE
Global speed control

### DISCUSSION THREAD

**Discussion by VegaDeftwing** [noisedeck/noisedeck-docs#18] [Ideas]
I've found a few times that I just want to slow everything down, but the sliders causing things to be fast aren't exactly obvious. Having a sort of global speed slider would be really nice. 
I don't know how the code looks to know if this is even practical, but maybe if it can't be done easily directly, slowing down via interpolation might be possible? It would probably act as a weird effect in its own right at extreme values too.

**Comment by jesshewitt** [comment]
We've had a few people ask for this and will add it to the feature requests with a higher priority. 

In the meantime, if you have the pro version, the easiest way to slow everything down is to make the animation loop longer, so that it's performing the same animation over, say, 5 seconds instead of the default 1.5. We might change how this works in the future, but it works for now.

In both free and pro, the per-module power and loop power settings have some impact on the apparent speed, with values closer to 0 appearing to be slower.

**Reply by VegaDeftwing** [reply]
oh, hmm, I didn't actually realize just making the animation loop longer worked, I had assume that would just increase the number of loops. It's actually basically exactly the behavior I'd want, so just exposing it as a slider might work?

**Reply by VegaDeftwing** [reply]
For S&G's I set it to 8s and 3840x2160 with Video feedback, Newton, Abberation, and Lens. That render took a solid 15 minutes or so on my Ryzen 1700 and Vega 56, but it did finish. It did manage to cause my system (Linux) to hang for a second as it tried to eat my entire GPU, but it did recover. I'm super impressed you've gotten something browser based to handle this. End file was 223.5 MB

**Reply by jesshewitt** [reply]
Haha, I'm surprised that it worked too. Trying to do anything over like 1024x1024 brings my aging gaming laptop to its knees, so I wouldn't even dare something like that.
